Library/Formula/ansible-cmdb.rb

```diff
@@ -16,11 +16,13 @@ class AnsibleCmdb < Formula
 
   def install
     bin.mkpath
+    man1.mkpath
     inreplace "Makefile" do |s|
       s.gsub! "/usr/local/lib/${PROG}", prefix
       s.gsub! "/usr/local/bin", bin
+      s.gsub! "/usr/local/share/man/man1/", man1 + 'ansible-cmdb.1'
+      s.gsub! "mandb", "#mandb"
```

Comment:
… but sadly this isn't possible that easily:

The released ZIP archive is generated by the toplevel `Makefile` in the source tree, but includes `./contrib/release_Makefile` (which fboender/ansible-cmdb#49 and fboender/ansible-cmdb#50 work on) as `./Makefile` – so the patches don't apply to the sources in the ZIP archive :-(

So I think it is best to keep these two lines in the Formula file for now …